src/SelectedWallet/WalletSelection/WalletSelectionScreen.tsx

```diff
@@ -209,6 +179,67 @@ const OnlyDevButton = () => {
   return <Button onPress={() => navigation.navigate('screens-index')} title="Dev options" style={styles.button} />
 }
 
+const useOpenWallet = (options?: UseMutationOptions<void, Error, WalletMeta>) => {
```

Comment:
hooks should be free from dependencies
make a simple ~~useCloseWallet~~ useOpenWallet hook that doesnt have any other hooks in it
then pass whatever you need in as parameters

Reply:
they need to be reusable in yoroi-extension